app/brave_settings_strings.grdp

```diff
@@ -1190,19 +1190,13 @@
   <message name="IDS_SETTINGS_IPFS_AUTO_FALLBACK_TO_GATEWAY_DESC" desc="The description for the switch of automatically fallback to IPFS public gateway">
     Automatically fall back to public IPFS gateway if your Brave local IPFS node cannot be reached.
   </message>
-  <message name="IDS_SETTINGS_IPFS_AUTO_REDIRECT_GATEWAY_LABEL" desc="The text label for redirecting IPFS resources to the configured gateway">
-    Redirect IPFS resources to the configured IPFS gateway
+  <message name="IDS_SETTINGS_IPFS_AUTO_REDIRECT_TO_GATEWAY_WHEN_POSSIBLE_LABEL">
+    Automatically redirect resources available on IPFS network to the configured gateway.
```

Comment:
Suggestion: `Automatically redirect requests for IPFS network resources to the configured gateway.` cc: @rmcfadden3